src/main/java/Duke.java

```diff
@@ -1,10 +1,261 @@
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    public static void main(String[] args) throws DukeException, FileNotFoundException {
+        Database database = new Database( "data.txt");
+        ArrayList<String> listOfTasks;
+        ArrayList<Task> myList;
+       try {
+            listOfTasks = database.readFile();
+            myList = readInput(listOfTasks);
+        }
+        catch (FileNotFoundException e){
```

Comment:
The ``catch`` keyword should be in line with the closing brace.

```suggestion
        } catch (FileNotFoundException e){
```